launch-scripts/launch_gapbs_gem5_20.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env python3
+
+# This is a job launch script for GAPBS tests with gem5-20
+
+import os
+import sys
+from uuid import UUID
+from itertools import starmap
+from itertools import product
+
+from gem5art.artifact.artifact import Artifact
+from gem5art.run import gem5Run
+from gem5art.tasks.tasks import run_gem5_instance
+
+packer = Artifact.registerArtifact(
+    command = '''wget https://releases.hashicorp.com/packer/1.4.3/packer_1.4.3_linux_amd64.zip;
+    unzip packer_1.4.3_linux_amd64.zip;
+    ''',
+    typ = 'binary',
+    name = 'packer',
+    path =  'disk-image/packer',
+    cwd = 'disk-image',
+    documentation = 'Program to build disk images. Downloaded sometime in August from hashicorp.'
+)
+
+experiments_repo = Artifact.registerArtifact(
+    command = 'git clone https://github.com/darchr/gem5art-experiments.git',
+    typ = 'git repo',
+    name = 'gapbs-test',
+    path =  './',
+    cwd = '../',
+    documentation = 'main experiments repo to run GAPBS with gem5'
+)
+
+gem5_repo = Artifact.registerArtifact(
+    command = 'git clone https://gem5.googlesource.com/public/gem5',
+    typ = 'git repo',
+    name = 'gem5',
+    path =  'gem5/',
+    cwd = './',
+    documentation = 'cloned gem5 from googlesource and checked out release-staging-v20.0.0.0 (May 6th, 2020)'
+)
+
+m5_binary = Artifact.registerArtifact(
+    command = 'make -f Makefile.x86',
+    typ = 'binary',
+    name = 'm5',
+    path =  'gem5/util/m5/m5',
+    cwd = 'gem5/util/m5',
```

Comment:
@powerjg, How can we make the m5 binary in gem5-20?

Reply:
<evil laugh>.... :( Bring this up at the meeting today, please :)

short answer:
```
cd util/m5
scons build/x86/out/m5
```

Long answer:
https://www.mail-archive.com/gem5-dev@gem5.org/msg34225.html